mitodl/ocw-hugo-themes
# add course collection display

#### Pre-Flight checklist


#### What are the relevant tickets?

no ticket over here

#### What's this PR do?

This does basically two things:

- rename our existing templates and code to reflect renaming 'course collections' to 'course lists'
- add rendering of the new, list-of-lists which is what we're now calling a 'course collection'

#### How should this be manually tested?

You'll need to get ahold of some course list and course collection content. The easiest way right now I think is to check out https://github.com/mitodl/ocw-studio/pull/1146 and then get the content out of that. If you have a github organization set up on MIT github you can then clone your `ocw-www` repo locally and set `WWW_CONTENT_PATH` to be the path to that repo. Then you could be able to check out rendered course lists (at `/course-lists/$SLUG`) and collections (at `/collections/$SLUG`).

#### Screenshots (if appropriate)

<img width="1652" alt="Screen Shot 2022-03-21 at 2 33 14 PM" src="https://user-images.githubusercontent.com/6207644/159340685-c5e9acd6-71ac-471c-93f6-56f04a6545d2.png">
